modules/web/src/app/node-data/component.ts

```diff
@@ -124,6 +127,10 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
     return this._clusterSpecService.cluster.spec.version < this.endOfDynamicKubeletConfigSupportVersion;
   }
 
+  get displayQuotaInWizard(): boolean {
+    return this.isEnterpriseEdition && (this.wizardMode === WizardMode.CustomizeClusterTemplate || !this.wizardMode);
```

Comment:
IMO, keeping `isEnterpriseEdition` in template is more explicit + Adding this WHEN to show quota should be dealt in this getter because we only want to show QuotaWidget only

**Show Widget**
- CustomizeClusterTemplate
- Create by wizard

Other than
- Edit / Customize. -> We don't want to show template that will be more clear from CODE
thoughts? this is my opinion